RTech.CoffeeMachine.Api/BrewCoffee/Controllers/CoffeeMachineController.cs

```diff
@@ -12,54 +12,48 @@ public class CoffeeMachineController : ControllerBase
     private readonly ILogger<CoffeeMachineController> _logger;
     private readonly IDateTimeProvider _dateTimeProvider;
     private readonly IBrewStatusProvider _brewStatusProvider;
-    // Interviewer note: MemoryCache from Microsoft.Extensions.Caching.Abstractions is thread safe
-    private readonly IMemoryCache _cache;
-    public static readonly string CACHE_KEY_NUM_REQUESTS = $"{nameof(CoffeeMachineController)}:numRequests";
```

Comment:
This code (including the code comment) was moved to the new filter file (see `BrewStatusUnavailableFilter.cs`)